### DISCUSSION TITLE
Manager node with search index

### DISCUSSION THREAD

**Discussion by xSunShine** [Security-Onion-Solutions/securityonion#9756] [General] [answered]
When i create manager node with search with node and it says other search nodes optional, does it mean that i install manager with search node and if i want i can add new ones latter throw standard proccedure? just want to be sure.

**Comment by InfosecGoon** [comment, marked as answer]
Yes. If you create a Manager/Search node, that will fill both roles as a manager and a search node. If you want to add more storage later, you can scale horizontally by adding more Search nodes to the grid.

If you're going to make a Manager/Search, make sure that it has sufficient disk space to handle both roles.

https://docs.securityonion.net/en/2.3/hardware.html

**Reply by xSunShine** [reply]
Thank you i tested it for couple of hours and it works great,Thank you
